lib/asciidoctor/latex/converter.rb

```diff
@@ -328,7 +328,7 @@ def inline_anchor node
           refs = node.parent.document.references[:ids]
           # FIXME: the next line is HACKISH (and it crashes the app when refs[refid]) is nil)
           # FIXME: and with the fix for nil results is even more hackish
-          if refs[refid]
+          if not node.text and refs[refid]
```

Comment:
This logic is correct.

I would prefer the syntax to be:

```
if !node.text && refs[refid]
```

but I realize that the project uses "not" and "and" in a lot of places...so that's more of a general request.